code/modules/reagents/reactions/reaction_recipe_food.dm

```diff
@@ -9,7 +9,10 @@
 	var/location = get_turf(holder.get_reaction_loc(chemical_reaction_flags))
 	if(obj_result && isturf(location))
 		for(var/i = 1, i <= created_volume, i++)
-			new obj_result(location)
+			create_food(location)
+
+/decl/chemical_reaction/recipe/food/proc/create_food(location)
+	return new obj_result(location)
```

Comment:
actually come to think of it maybe this should use dropInto, downstream i want cooking containers to contain stuff made in them and they're valid reagent containers + the stove is a heat source... until that's replaced by something downstream i'd like produced food to stay in the container and imo using dropinto is a good way to handle it